share/zfsnap/commands/send.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/sh
+
+# This file is licensed under the BSD-3-Clause license.
+# See the AUTHORS and LICENSE files for more information.
+
+SSH_KEY=~/.ssh/id_rsa
+#EMAIL=none@none
+user=$USER
+LOCAL='false'
+FULL='false'
+REVERSE='false'
+
+# FUNCTIONS
+Help() {
+    cat << EOF
+${0##*/} v${VERSION}
+
+Syntax:
+${0##*/} destroy [ options ] zpool/filesystem ...
+
+OPTIONS:
+  -b           = source (to be backed up) zfs pool and filesystem, pool_name/filesystem_name
+  -d           = destination (to store backup) zfs pool and filesystem, pool_name/filesystem_name
+  #-e		   = email address to send notification to
+  #-f 		   = enable sendin full backup rather than incremental
+  -h           = Print this help and exit
+  -i		   = remote zfs systems IP address or DNS name
+  #-l		   = indicates this will be a backup to a local zfs file system and ssh is not needed
+  -k 		   = ssh key to be used to authenticate to remote zfs system; default ~/.ssh/id_rsa
```

Comment:
Perhaps expanding this to any options to pass to `ssh`, as many people like playing with ciphers, alternate ports, etc.